algoliasearch/src/main/java/com/algolia/search/saas/BaseAPIClient.java

```diff
@@ -62,7 +62,9 @@
     private List<String> readHosts;
     private List<String> writeHosts;
 
-    /** HTTP headers that will be sent with every request. */
+    /**
+     * HTTP headers that will be sent with every request.
+     */
```

Comment:
I'm not sure to see the issues here:
1. can be addressed by having a first commit reformatting, then reviewing *the other commits*, then squashing when satisfied
2. To me it's just one more change but you can still blame:
<img width="1027" alt="git blame after formatting" src="https://cloud.githubusercontent.com/assets/1821404/14352589/8e6d1634-fcd5-11e5-9bc2-6b835317e459.png">